healthz/defaults/defaults.go

```diff
@@ -0,0 +1,4 @@
+package defaults
+
+// AccessKeyParam is a name of access key query param
+const AccessKeyParam = "accesskey"
```

Comment:
I don't see any other option to share this value between this:
https://github.com/gravitational/satellite/pull/5/files#diff-200368289b120e00f70756542623dcdaR30
and this:
https://github.com/gravitational/satellite/pull/5/files#diff-63cc16d8767708eccc0a0dbef34c84a9R32

any advices?